Made code asynchronous, use rich, some other stuff

TheOnlyWayUp opened the pull request:
Please go through the code, I'm not fully done, oldStats is an example of something that's incomplete, status should also be set in the bot class. pic.twitter.com/oUK4DYbUkQ for example. Also, I suggest you rewrite the code using on_command_error for proper usage instead of checking if an argument is a default value, using aiohttp and aiofiles. I need to go now, please go through it and lmk if anything needs to be changed, you have push access to this branch.

TheOnlyWayUp committed:
Update bot.py

Aniell4 reviewed (changes requested):
"olderStats" variable defined but never used, program should error at line 214, "stats" is not defined.

gvmii committed:
Update bot.py

ASYNCAWAITASYNCAWAITASYNCAWAITASYNCAWAITASYNCAWAITASYNCAWAITASYNCAWAITASYNCAWAITASYNCAWAIT

Ixogamer commented:
should be working now :)

Ixogamer commented:
Thanks a lot again for contributing, you don't know how much I appreciate it :)
